lib/oli_web/controllers/api/xapi_controller.ex

```diff
@@ -22,6 +22,8 @@ defmodule OliWeb.Api.XAPIController do
 
       {:error, e} ->
         Logger.error("Error constructing xapi bundle: #{inspect(e)}")
+        Oli.Utils.Appsignal.capture_error("Error constructing xapi bundle: #{inspect(e)}")
```

Comment:
It seems like there should be some way to notify AppSignal any time Logger.error is called, so that we don't need to explicitly call it separately. But that is something we can maybe explore in a non-hotfix release